docs/reference/method/MongoDBCollection-createIndexes.txt

```diff
@@ -9,78 +9,72 @@ MongoDBCollection::createIndexes()
    :backlinks: none
    :depth: 1
    :class: singlecol
-                    
+
 Definition
 ----------
 
 .. phpmethod:: MongoDB\\Collection::createIndexes($indexes)
-               
+
    Create one or more indexes for the collection.
-   
+
    .. code-block:: php
 
-      function createIndexes(array $indexes): string[]
-
-   ``createIndex()`` has one parameter: ``$indexes``, which is an array
-   or object.
-   Each element in the ``$indexes`` array|object must have a
-   "``key`` document" that specifies the fields to index and the
-   array direction or type. You can then specify index options as
-   additional key-value pairs in that element.
-
-   For example, the following ``$indexes`` parameter creates an ascending
-   unique index on the ``username`` field:
-   
-   .. code-block:: none
-                   
-      [ 'key' => ['username => 1 '], 'unique' => true ],
-   
-   The following ``$indexes`` parameter creates a 2dsphere index on the ``loc`` field and
-   specifies a custom index name:
-   
-   .. code-block:: none
-                   
-      [ 'key' => [ 'loc' => '2dsphere'], 'name' => 'geo_index' ],
-   
-   You can specify any index options that your MongoDB version
-   supports. MongoDB 3.2 includes the following options:
-                
-   .. include:: /includes/apiargs/MongoDBCollection-method-createIndex-option.rst
```

Comment:
After adding the actual parameter and options includes for this method, which differ from `createIndex()`, I tried moving the general index options into a section below entitled "$indexes parameter". I found that the table didn't render the same as this "Definition" section (with leading indentation). The second paragraph of each option description would increase in font size, which is likely a CSS issue.

Ultimately, I decided not to repeat the index options and instead link to `createIndex()` and the MongoDB manual. I feel this also works better because `writeConcern` is one of the `createIndex()` method options despite the fact that it's not an index option. Maintaining a separate include of just index options _and_ `writeConcern` seemed like a fair bit of overhead. As-is, explaining how `$indexes` works in its own section below makes `createIndexes()` method docs easier to digest, since we only discuss the actual params/options in the "Definition" section.